modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringBootServerCodegen.java

```diff
@@ -119,9 +131,6 @@ public void addOperationToGroup(String tag, String resourcePath, Operation opera
         if (basePath == "") {
             basePath = "default";
         } else {
-            if (co.path.startsWith("/" + basePath)) {
-                co.path = co.path.substring(("/" + basePath).length());
-            }

```

Comment:
And this is exactly what you do in your interface implementation :smile: (except that you put back in the template the path segment that was removed by the codegen) !